flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/state/rocksdb/RocksDBStateUploaderTest.java

```diff
@@ -233,6 +238,28 @@ void testMultiThreadUploadCorrectly() throws Exception {
         }
     }
 
+    @Test
+    void testApplyJitter() throws Exception {
+        try (RocksDBStateUploader rocksDBStateUploader =
+                new RocksDBStateUploader(1, Duration.ofMillis(1000L))) {
+            rocksDBStateUploader.applyJitter(
+                    sleepTime -> {
+                        assertThat(sleepTime).isGreaterThan(0);
+                        assertThat(sleepTime).isLessThanOrEqualTo(1000);
+                    });
+        }
+    }
+
+    @Test
+    void testApplyDefaultJitter() throws Exception {
+        try (RocksDBStateUploader rocksDBStateUploader =
+                new RocksDBStateUploader(1, Duration.ZERO)) {
+
+            // If it is configured as Zero, the consumer function shouldn't be called
```

Comment:
nit: shouldn't be -> isn't

Reply:
The "shouldn't" is to explain why apply jitter shouldn't be called otherwise the test case will fail.